report/src/main/webapp/app/components/keyboardShortcuts.directive.js

```diff
@@ -141,36 +142,36 @@ define(['angularAMD', 'userSettingsService'], function (angularAMD) {
             if (!_.isEmpty($(suiteContainer).nextAll(
                     '.aside-report:not(.is-hidden)').first().find(
                     '.test-name'))) {
-              testContainer.removeClass('is-active');
-              currentItem.removeClass('is-active');
-              toggleNextTest(suiteContainer);
+              currentTabLabel = $('.nav-tabs > .nav-item').filter('.active').text().replace(/\s/g, '');
```

Comment:
This line is repeated across the file few times - can it be moved to a seperate function, e.g. `getCurrentTabLabel`?

Reply:
Fixed.